### PR TITLE
Even more terra-fying

### PR DESCRIPTION
**Pull request by RoriCremer**
Screen Shot 2019-04-04 at 11 02 34 PM

Screen Shot 2019-04-05 at 2 13 28 PM

### CODE REVIEW

File: src/components/DatasetView.jsx [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
-import { Link, NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
 
 import { getDatasets } from 'actions/index';
-import JadeTable from './table/JadeTable';
-import AddSVG from '../../assets/media/icons/plus-circle-solid.svg';
+import DatasetTable from './table/DatasetTable';
 
 const styles = theme => ({
```
Comment (ddietterich) [inline src/components/DatasetView.jsx:9]: Is something like this shareable? It looks like these style thingies are the same or very similar to ones above.

**Review by ddietterich — COMMENTED** [review]
I kinda get what this is doing, but only kinda. One question for my education.

File: src/components/table/StudyTable.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        render: row => <Link to={`/studies/details/${row.id}`}>{row.name}</Link>,
+      },
+      {
+        label: 'Description',
+        property: 'description',
+      },
+      {
+        label: 'Last changed',
+        property: 'modifiedDate',
+        render: row => moment(row.createdDate).fromNow(),
```
Comment (ahaessly) [inline src/components/table/StudyTable.jsx:45]: should this be row.modifiedDate?

File: src/sagas/repository.js [hunk 3]
```diff
@@ -164,7 +161,7 @@ export function* addReaderToDataset({ payload }) {
   const datasetId = payload.datasetId;
   try {
     const response = yield call(
-      authGet,
+      authPost,
```
Comment (ahaessly) [inline src/sagas/repository.js:164]: where do you pass the payload?

**Review by ahaessly — APPROVED** [review]
2 comments/questions

File: src/components/table/StudyTable.jsx (hunk 2 above)
Comment (thathert) [reply src/components/table/StudyTable.jsx:45]: It will be eventually -- we need to add modified dates to the API.

**Review by thathert — APPROVED** [review]
LGTM -- will need to add payloads to authPost

**Pull request merged**